packages/miniflare/src/workers/assets/assets-kv.worker.ts

```diff
@@ -17,9 +17,7 @@ export default <ExportedHandler<Env>>{
 			return new Response(message, { status: 405, statusText: message });
 		}
 
-		// don't uri decode pathname, because we encode the filepath before hashing
 		const pathHash = new URL(request.url).pathname.substring(1);
-
 		const entry = env.ASSETS_REVERSE_MAP[pathHash];
```

Comment:
Yep, this is after the asset worker has gotten the pathname **with** leading /, and hashed it. In dev, miniflare sets up the `assets-kv-worker` which gets passed this pathHash and look it up in the reverse map to get it from the filesystem instead of an actual kv namespace. And the leading '/' in this file is not actually part of the pathHash so its discarded.


/some-path -> asdfasddsfdasf-pathhash -> /asdfasddsfdasf-pathhash -> asdfasddsfdasf-pathhash